crates/store/re_types_core/src/tuid.rs

```diff
@@ -11,13 +11,18 @@ use crate::{DeserializationError, Loggable};
 
 // ---
 
+// TODO(emilk): This is a bit ugly… but good enough for now?
+pub fn tuid_arrow_fields() -> Fields {
```

Comment:
Can't you get rid of this if you get rid of using the builder type for an empty array?

Reply:
Yes, but at the cost of some ugly dynamic casting and unwraps that I'd rather avoid